### PR TITLE
about us section now leads to about us page on click

### PR DESCRIPTION
**Pull request by incalescence**

### CODE REVIEW

**Review by imagine-hussain — APPROVED** [review]
small import fix;
otherwise lgtm!

File: src/routes/LandingPage/AboutUsSection/AboutUsSection.jsx [hunk 1]
```diff
@@ -3,36 +3,42 @@ import './index.less';
 
 import { Typography } from 'antd';
 import placeholder from '../../../assets/about_us_placeholder.png';
+import { Router } from 'react-router-dom';
+import { Link } from 'react-router-dom';
```
Comment (imagine-hussain) [inline src/routes/LandingPage/AboutUsSection/AboutUsSection.jsx:7]: This should just destructure one import
```suggestion
import { Link, Router } from 'react-router-dom';
```

**Comment by imagine-hussain** [comment]
btw, i think there was some miscommunication on our end on what porting over stuff meant, will clarify this in meeting

**Comment by incalescence** [comment]
Updated the actual about page

**Pull request merged**